cmd/reva/import.go

```diff
@@ -40,133 +64,115 @@ func importCommand() *command {
 			fmt.Println(cmd.Usage())
 			os.Exit(1)
 		}
-		f := cmd.Args()[0]
+		exportPath := cmd.Args()[0]
 
-		sClient, err := getStorageProviderClient()
-		if err != nil {
-			return err
-		}
-		uClient, err := getUserShareProviderClient()
-		if err != nil {
-			return err
-		}
+		return importShares(exportPath)
```

Comment:
`importShares` returns an error. Could you handle the error here instead? not within the function

```go
err := importShares(exportPath)
if err != nil {
  return err
}

return nil